components/twitter/sources/language-codes.js

```diff
@@ -1,4 +1,8 @@
 module.exports = [
+  {
+    English: "Undefined",
```

Comment:
Would it make sense to use something more user friendly as the english label for `Undefined`?

Reply:
It definitely does make sense, I just don't know what to use here 😂 
The [Twitter API docs](https://developer.twitter.com/en/docs/twitter-api/enterprise/search-api/guides/operators) use the term **undefined** so I just used it here as well, but we can change it to **Undetected** maybe?

Reply:
Does `Any` make sense?

Reply:
Well... that would imply that this option captures tweets in any language, when in fact what it does is capture those tweets which language couldn't be properly identified by Twitter. WDYT?